examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
ok just to point out how semi crazy this is to "properly" ignore jquery this is what i had to do

```
node_modules/jquery/.bowerrc
node_modules/jquery/.npmignore
node_modules/jquery/AUTHORS.txt
node_modules/jquery/CONTRIBUTING.md
node_modules/jquery/MIT-LICENSE.txt
node_modules/jquery/README.md
node_modules/jquery/bower.json
node_modules/jquery/package.json
node_modules/jquery/.jscsrc
node_modules/jquery/src/*
node_modules/jquery/dist/*
!node_modules/jquery/dist/jquery.js
```
and that is for a semi well organized lib 

this should is a responsibility of package author in the npm ignores .. not us, I know you think it will make upgrades easier but take this example.. if an author decides to reorganize any of their app the ignores are now invalidated and have to be redone.

Its cool tho I will finish out this example just to show you, however I stand strong behind `git add -f`